device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransportConnection.java

```diff
@@ -24,11 +24,9 @@
 {
     /**
      * Opens the transport connection object
-     * @param deviceClientConfigs The list of configs to use. If more than 1 configs are in this list, multiplexing
-     *                            will be used
      * @throws TransportException If any exceptions are encountered while opening the connection
      */
-    void open(Queue<DeviceClientConfig> deviceClientConfigs) throws TransportException;
```

Comment:
It never made sense that we defined the possibility of multiplexing in this interface considering only AMQP supports it, so I've removed it

Reply:
Not a customer-breaking change?